php/libraries/SinglePointLogin.class.inc

```diff
@@ -463,37 +463,29 @@ class SinglePointLogin
     ): int {
         $factory = NDB_Factory::singleton();
         $db      = $factory->database();
-        $query   = "SELECT Login_timestamp from user_login_history " .
+        $query   = "SELECT COUNT(loginhistoryID)" .
```

Comment:
could use "SELECT COUNT(loginhistoryID) as 'count' ...  to simplify retrieval of data from the array
as well as pselectOne as you will alway get only 1 value